### ISSUE TITLE
[Feature Request] Set RedirectUri inside AddMicrosoftIdentityWebApp()

### ISSUE THREAD

**Issue by rastrup** [AzureAD/microsoft-identity-web#2547]
**Is your feature request related to a problem? Please describe.**
If I use AppBuilderExtension.AddMicrosoftIdentityWebApp() with an app registration containing multiple Uri's it will just select one of these Uri's for redirection after successful login. 
AddMicrosoftIdentityWebApp does not read the value RedirectUri from any setting.
I have to add a custom updateOptions to the method call that sets the RedirectUri.

`app.AddMicrosoftIdentityWebApp(factory, updateOptions: UpdateOptions);

private static void UpdateOptions(OpenIdConnectAuthenticationOptions options)
{
    options.RedirectUri = "MyRedirectUri";
}`

**Describe the solution you'd like**
I would like for AddMicrosoftIdentityWebApp to read the RedirectUri from settings (most likely appsettings.json) and use that value if available.

**Comment by jennyf19** [comment]
@rastrup Have you considered moving to .NET 8?

**Comment by rastrup** [comment]
> @rastrup Have you considered moving to .NET 8?

Yes, the client has considered it but due to the technical debt in the solution found that now is not the right time.
But what has that to do with my suggestion?
